### PR TITLE
PR proposal test

### PR DESCRIPTION
**Pull request by anoadragon453**
This is a really exciting proposal

### CODE REVIEW

**Comment by anoadragon453** [comment]
I think this is prettyneat.jpg

@mscbottest2 fcp merge

**Comment by anoadragon453** [comment]
That might help

@mscbottest2 fcp merge

**Comment by mscbottest2** [comment]
**This FCP proposal has been cancelled by https://github.com/mscbottestorg/faketrix-doc/pull/2#issuecomment-703658558.**

Team member @mscbottest2 has proposed to **merge** this. The next step is review by the rest of the tagged people:

- [x] @anoadragon453
- [ ] @mscbottest2


Concerns:

* please


Once at least 75% of reviewers approve (and there are no outstanding concerns), this will enter its final comment period. If you spot a major issue that hasn't been raised at any point in this process, please speak up!

See [this document](https://github.com/matrix-org/mscbot-python/blob/master/docs/user_guide.md) for information about what commands tagged team members can give me.

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-bla
+bla2
```
Comment (anoadragon453) [inline README.md:1]: Actually, I don't this *is* all that neat.

@mscbottest2 concern Is that neat?

Comment (anoadragon453) [reply README.md:1]: Idk

Comment (anoadragon453) [reply README.md:1]: @mscbottest2 concern what do you think?

Comment (anoadragon453) [reply README.md:1]: @mscbottest2 concern please

Comment (anoadragon453) [inline README.md:1]: What about starting from scratch?

@mscbottest2 fcp merge

**Review by anoadragon453 — COMMENTED** [review]
lgtm

**Comment by mscbottest2** [comment]
An FCP proposal is already in progress.

**Comment by anoadragon453** [comment]
@mscbottest2 fcp cancel

**Comment by anoadragon453** [comment]
@mscbottest2 fcp cancel

**Comment by anoadragon453** [comment]
@mscbottest2 fcp cancel

**Comment by anoadragon453** [comment]
@mscbottest2 fcp merge

**Comment by mscbottest2** [comment]
Team member @mscbottest2 has proposed to **merge** this. The next step is review by the rest of the tagged people:

- [x] @anoadragon453
- [x] @mscbottest2


Concerns:

* ~~I have a concern about how concerning concerns are. This concerns me.~~
* ~~I have a concern about how concerning concerns are. This concerns me!~~


Once at least 75% of reviewers approve (and there are no outstanding concerns), this will enter its final comment period. If you spot a major issue that hasn't been raised at any point in this process, please speak up!

See [this document](https://github.com/matrix-org/mscbot-python/blob/master/docs/user_guide.md) for information about what commands tagged team members can give me.

**Comment by anoadragon453** [comment]
@mscbottest2 concern I have a concern about how concerning concerns are. This concerns me.

**Comment by anoadragon453** [comment]
@mscbottest2 concern I have a concern about how concerning concerns are. This concerns me!

**Comment by anoadragon453** [comment]
@mscbottest2 resolve I have a concern about how concerning concerns are. This concerns me.

@mscbottest2 resolve I have a concern about how concerning concerns are. This concerns me!

**Comment by mscbottest2** [comment]
:bell: This is now entering its final comment period, as per [the review](https://github.com/mscbottestorg/faketrix-doc/pull/2#issuecomment-703658713) above. :bell:

**Comment by anoadragon453** [comment]
@mscbottest2 fcp cancel

**Comment by mscbottest2** [comment]
Final comment period for this proposal has been cancelled.

**Comment by anoadragon453** [comment]
@mscbottest2 concern I have a concern about how concerning concerns are. This concerns me!

**Comment by mscbottest2** [comment]
That concern has already been raised

**Comment by anoadragon453** [comment]
@mscbottest2 concern I have a concern about how concerning concerns are. This concerns me!

**Comment by mscbottest2** [comment]
That concern has already been raised

**Pull request closed without merging**